lib/graphql/cache/field.rb

```diff
@@ -7,20 +7,20 @@ class Field < ::GraphQL::Schema::Field
       def initialize(
         *args,
         cache: false,
-        expiry: nil,
         **kwargs,
         &block
       )
-        @cache_config = {
-          cache: cache,
-          expiry: expiry
-        }
+        @cache = if cache.is_a? Hash
+                   cache
+                 else
+                   { cache: cache }
```

Comment:
This is a touch misleading.  We should probably change it to:

```ruby
if cache.is_a? Hash
  cache
else
  { cache: true }
end
```
since we're interpreting any non-nil value as enabling the cache when `cache` isn't a hash.